custom_components/tahoma/switch.py

```diff
@@ -64,19 +64,23 @@ async def async_turn_on(self, **_):
             await self.async_execute_command(COMMAND_ON)
 
         elif self.has_command(COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE):
-            # Values taken from iosiren.js (tahomalink.com). Parameter usage is currently unknown.
             await self.async_execute_command(
-                COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE,
-                120000,
-                75,
-                2,
+                COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE,  # https://www.tahomalink.com/enduser-mobile-web/steer-html5-client/vendor/somfy/io/siren/const.js
+                2 * 60 * 1000,  # 2 minutes
+                75,  # 90 seconds bip, 30 seconds silence
+                2,  # repeat 3 times
                 COMMAND_MEMORIZED_VOLUME,
             )
 
     async def async_turn_off(self, **_):
         """Send the off command."""
+        if self.has_command(COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE):
+            return await self.async_execute_command(
+                COMMAND_RING_WITH_SINGLE_SIMPLE_SEQUENCE, 2000, 100, 0, "standard",
+            )
+
         if self.has_command(COMMAND_OFF):
-            await self.async_execute_command(COMMAND_OFF)
+            return await self.async_execute_command(COMMAND_OFF)
```

Comment:
async_turn_off does not return anything.

Reply:
True, I used return to make sure only one of the command would be used. I could move to if / elseif, but this is how we do it in covers.py as well.

Reply:
Fixed